plotly/tests/test_core/test_tools/test_validate.py

```diff
@@ -8,28 +8,28 @@
 
 def test_validate_valid_fig():
     fig = {
-        'layout':{
-            'title':'something'
+        'layout': {
+            'title': 'something'
         },
-        'data':[
+        'data': [
             {
-                'x':[1,2,3],
-                'y':[2,1,2]
+                'x': [1, 2, 3],
+                'y': [2, 1, 2]
             }
         ]
     }
     tls.validate(fig, 'Figure')
 
 
-@raises(PlotlyError)
+@raises(ValueError)
```

Comment:
🚨 Breaking Change

Reply:
FYI. So far I haven't incorporated any of the legacy plotly-specific exceptions types. If we want to do this we'll need a bit of an inventory of which exceptions we want to keep around for what purposes.